packages/matchbox/src/components/Tag/styles.js

```diff
@@ -0,0 +1,150 @@
+import { tokens } from '@sparkpost/design-tokens';
+
+export const tagBase = props => `
```

Comment:
<img width="534" alt="Screen Shot 2020-02-04 at 4 07 18 PM" src="https://user-images.githubusercontent.com/3613392/73787040-726ae380-4768-11ea-9e1e-bb46572ad272.png">

I know this has been an eternal struggle with this component, but is this not vertically centered? Some line-height creeping in here?

Reply:
Let me see if I can fix this, I think this slight offset is coming from the icons `vertical-align: middle`

Reply:
Much better. Looks good.